content/features/featuresDeepDive/animation/baked_texture_animations.md

```diff
@@ -99,6 +99,32 @@ Here's an example:
 
 <Playground id="#CP2RN9#20" title="Vertex Texture Animations on thin instances" description="An example of playing VATs on thin instances."/>
 
+### VATs offset
+
+When using VAT with thin instances, you might expect each instance to start its animation with a specific time
+offset. However, this is **not guaranteed by default**.
+
+This is because multiple instances often share the same `BakedVertexAnimationManager`, which internally handles the
+current animation frame globally. To ensure that a specific instance starts at its intended offset you can manually
+compute the correct offset using the following formula:
+
+```typescript
+function computeOffset(
+  fromFrame: number,
+  toFrame: number,
+  time: number,
+  fps: number = 60
+): number {
+  const totalFrames = toFrame - fromFrame + 1;
+  const t = time * fps / totalFrames;
+  const frame = Math.floor((t - Math.floor(t)) * totalFrames);
+  return totalFrames - frame;
+}
+```
+
```

Comment:
I think you should indicate that `time` should be the time of the BVA manager at which `computeOffset()` is called (`BakedVertexAnimationManager.time`), as this may not be obvious to the reader.

Reply:
done